app/models/chat_bot.rb

```diff
@@ -0,0 +1,3 @@
+class ChatBot < ApplicationRecord
```

Comment:
@DeboDevelop - Chatbot is a single word.

Reply:
@AdityaTiwari2102 Done.